openpype/hosts/maya/plugins/load/load_arnold_standin.py

```diff
@@ -84,6 +93,11 @@ def load(self, context, name, namespace, options):
             sequence = is_sequence(os.listdir(os.path.dirname(self.fname)))
             cmds.setAttr(standin_shape + ".useFrameExtension", sequence)
 
+            fps = float(version["data"].get("fps"))
+            if fps is None:
+                fps = get_current_fps()
+            cmds.setAttr(standin_shape + ".abcFPS", fps)
```

Comment:
it is equivalent to:
```suggestion
            fps = float(version["data"].get("fps"), get_current_fps())
            cmds.setAttr(standin_shape + ".abcFPS", fps)
```